nixos/roles/prometheus/default-alerts.nix

```diff
@@ -0,0 +1,229 @@
+      };
+
+      PrometheusNotConnectedToAlertmanager = {
+        expr = "prometheus_notifications_alertmanagers_discovered < 1";
+        annotations.description = "Prometheus cannot connect the alertmanager\n  VALUE = {{ $value }}\n  LABELS = {{ $labels }}";
+      };
+
+      PrometheusRuleEvaluationFailures = {
+        expr = "increase(prometheus_rule_evaluation_failures_total[3m]) > 0";
+        annotations.description = "Prometheus encountered {{ $value }} rule evaluation failures\n  VALUE = {{ $value }}\n  LABELS = {{ $labels }}";
+      };
+
+      PrometheusTemplateExpansionFailures = {
+        expr = "increase(prometheus_template_text_expansion_failures_total[3m]) > 0";
+        for = "0m";
+        annotations.description = "Prometheus encountered {{ $value }} template text expansion failures\n  VALUE = {{ $value }}\n  LABELS = {{ $labels }}";
+      };
+
+      PromtailRequestsErrors = {
+        expr = "100 * sum(rate(promtail_request_duration_seconds_count{status_code=~\"5..|failed\"}[1m])) by (namespace, job, route, instance) / sum(rate(promtail_request_duration_seconds_count[1m])) by (namespace, job, route, instance) > 10";
+        for = "15m";
+        annotations.description = "{{ $labels.job }} {{ $labels.route }} is experiencing {{ printf \"%.2f\" $value }}% errors";
+      };
+
+      PromtailFileLagging = {
+        expr = "abs(promtail_file_bytes_total - promtail_read_bytes_total) > 1e6";
+        for = "15m";
+        annotations.description = "{{ $labels.instance }} {{ $labels.job }} {{ $labels.path }} has been lagging by more than 1MB for more than 15m";
+      };
+
+      Filesystem80percentFull = {
+        expr = ''disk_used_percent{mode!="ro"} >= 80'';
+        for = "10m";
+        annotations.description = "{{$labels.instance}} device {{$labels.device}} on {{$labels.path}} got less than 20% space left on its filesystem";
+      };
+
+      FilesystemInodesFull = {
+        expr = ''disk_inodes_free / disk_inodes_total < 0.10'';
+        for = "10m";
+        annotations.description = "{{$labels.instance}} device {{$labels.device}} on {{$labels.path}} got less than 10% inodes left on its filesystem";
+      };
+
+      DailyTaskNotRun = {
+        expr = ''time() - task_last_run{state="ok",frequency="daily"} > (24 + 6) * 60 * 60'';
+        annotations.description = "{{$labels.host}}: {{$labels.name}} was not run in the last 24h";
+      };
+
+      TenMinutesTaskNotRun = {
+        expr = ''time() - task_last_run{state="ok",frequency=""} > (24 + 6) * 60 * 60'';
+        annotations.description = "{{$labels.host}}: {{$labels.name}} was not run in the last 24h";
+      };
+
+      TaskFailed = {
+        expr = ''task_last_run{state="fail"}'';
+        annotations.description = "{{$labels.host}}: {{$labels.name}} failed to run";
+      };
+
+      NixpkgsOutOfDate = {
+        expr = ''(time() - flake_input_last_modified{input="nixpkgs",host!="matchbox"}) / (60*60*24) > 7'';
```

Comment:
I think the exclusion of a host named "matchbox" is probably copied from nix-community infra, but not needed for srvos?